<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-R44QD4T" failures="4" tests="5" name="Test" time="49.95" errors="0" timestamp="29 Feb 2020 20:22:01 GMT">
  <testcase classname="com.tests.Excercise1Test" name="excercise1" time="35.903"/>
  <testcase classname="com.tests.Excercise2Test" name="excercise2" time="0.003">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.tests.Excercise2Test.excercise2(Excercise2Test.java:17)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:830)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- excercise2 -->
  <testcase classname="com.tests.Excercise3Test" name="excercise3" time="0.001">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.tests.Excercise3Test.excercise3(Excercise3Test.java:15)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:830)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- excercise3 -->
  <testcase classname="com.tests.Excercise4Test" name="excercise4" time="0.002">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.tests.Excercise4Test.excercise4(Excercise4Test.java:12)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:830)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- excercise4 -->
  <testcase classname="com.tests.Excercise5Test" name="excercise5" time="0.001">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.tests.Excercise5Test.excercise5(Excercise5Test.java:14)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:830)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- excercise5 -->
</testsuite> <!-- Test -->
